### PR TITLE
NO-ISSUE: split kube helpers file

### PR DESCRIPTION
**Pull request by RazRegev**
Since it is a large file that will only get bigger,
it will be better to split it to a sub package.

### CODE REVIEW

File: discovery-infra/test_infra/helper_classes/kube_helpers/__init__.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+"""
+kube_helpers.py provides infra to deploy, manage and install cluster using
```
Comment (osherdp) [inline discovery-infra/test_infra/helper_classes/kube_helpers/__init__.py:2]: module not existing :)

**Review by osherdp — CHANGES_REQUESTED** [review]

File: discovery-infra/test_infra/helper_classes/kube_helpers/__init__.py (hunk 1 above)
Comment (RazRegev) [reply discovery-infra/test_infra/helper_classes/kube_helpers/__init__.py]: ops

**Review by osherdp — APPROVED** [review]
/lgtm

**Pull request merged**